hackathon_site/event/test_api.py

```diff
@@ -4,6 +4,8 @@
 from rest_framework.test import APITestCase
 from hackathon_site.tests import SetupUserMixin
 from django.contrib.auth.models import Permission
+import json
```

Comment:
Minor, move these to the top of the file above the 3rd party imports (separated by a blank line)